### ISSUE TITLE
About sumo traffic

### ISSUE THREAD

**Issue by yumianhuli2** [lgsvl/simulator#779]
Are there any plans to integrate the sumo traffic flow?

**Comment by EricBoiseLGSVL** [comment]
@yumianhuli2 Yes, we are currently working on extending NPC control to SUMO and other traffic control applications.

**Comment by yumianhuli2** [comment]
> @yumianhuli2 Yes, we are currently working on extending NPC control to SUMO and other traffic control applications.

Great Work！It's expected to be released on what date？

**Comment by lokesku** [comment]
We dont have a fixed date yet. But we will release soon. Subscribe to our mailing list to be notified when we do so: https://www.svlsimulator.com/
